pkg/monitor/api/types_test.go

```diff
@@ -4,6 +4,8 @@
 package api
```

Comment:
Same `cleanup: remove redundant methods` -> `monitor:hubble: remove redundant methods in favor of Decode`